### PR TITLE
bug(top-news): updates the top news section with time magazine.

### LINKED ISSUE #74: Top news not updating
## Expected Behavior
Top news should be updating daily.

## Actual/Current Behavior
Top News is stuck on the same news articles.

### PR DESCRIPTION
**Pull request by JorgeLVilla**
## Changes
_List Changes Introduced by this PR_
1. Created a new function to fetch data for Top news section.
2. Fetches data for Time magazine and renders it on to top news section.
3. Created a new interface for Top news to pull in source of magazine from API.

## Purpose
This allows the top news section to be populated with different information than the today's top news section.

_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #74

### CODE REVIEW

File: modern-news-app/src/App.tsx [hunk 1]
```diff
@@ -38,9 +37,11 @@ const App = () => {
 
   const getTopNewsData = async (search: string) => {
     try {
-      const request = new EverythingNewsRequest({q: search});
-      const dataObject: ITopNewsResponse = await fetchNewsData(request);
+      const request = new TopNewsRequest({q: search, sources: "time"});
+      const dataObject: ITopNewsResponse = await fetchTopNewsData(request);
       setTopNewsData(dataObject);
+      console.log(dataObject)
```
Comment (Lap343) [inline modern-news-app/src/App.tsx:43]: forgot to remove the console.log. Minus 10 points

**Review by Lap343 — APPROVED** [review]
Everything other than the console.log looks good

File: modern-news-app/src/App.tsx (hunk 1 above)
Comment (JorgeLVilla) [reply modern-news-app/src/App.tsx:43]: Removing that now sorry sir

**Review by BlackCubes — APPROVED** [review]
Looks gewd!!

File: modern-news-app/src/util/fetchApi.ts [hunk 2]
```diff
@@ -23,3 +24,15 @@ export const fetchTodayNewsData = async (request: ITodayNewsRequest) => {
         console.error(error);
     }
 }
+
+export const fetchTopNewsData = async (request: ITopNewsRequest) => {
+    try {
+        // url for Time Magazine news site
+        const topNewsUrl = `https://newsapi.org/v2/top-headlines?sources=${request.sources}&apiKey=${request.apiKey}&pageSize=${request.pageSize}`;
+        console.log(topNewsUrl)
```
Comment (connielion) [inline modern-news-app/src/util/fetchApi.ts:32]: ooh

Comment (JayDampitan) [inline modern-news-app/src/util/fetchApi.ts:32]: You left another one here too Jorge. It happens.

Comment (JorgeLVilla) [reply modern-news-app/src/util/fetchApi.ts:32]: got rid of it sowy

**Review by connielion — APPROVED** [review]
Looks good. Just remove the 2 console logs.

**Review by JayDampitan — APPROVED** [review]
Beside the console.logs, looking good my friend!

**Pull request merged**